### PR TITLE
[2.19.x] DDF-5724 G-2115 fixes issue with basic search form when there are no facets

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
Fixes issue with basic search form when there are no facets
#### Ask 2 committers to review/merge the PR and tag them here.

Build / Install / Configure the Catalog UI to use a facet that will return nothing and verify one can still issue a query from the basic search form
#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by bdeining** [comment]
build now

**Comment by bdeining** [comment]
build now

File: ui/packages/catalog-ui-search/src/main/webapp/component/query-basic/query-basic.view.js [hunk 1]
```diff
@@ -543,7 +543,10 @@ module.exports = Marionette.LayoutView.extend({
     }
 
     const types = this.basicType.currentView.model.getValue()[0]
-    const specificTypes = this.basicTypeSpecific.currentView.model.getValue()[0]
+    const specificTypes =
+      this.basicTypeSpecific.currentView.model !== undefined
+        ? this.basicTypeSpecific.currentView.model.getValue()[0]
```
Comment (mojogitoverhere) [inline ui/packages/catalog-ui-search/src/main/webapp/component/query-basic/query-basic.view.js]: ❓ Can `getValue()` return an empty array?

Comment (bdeining) [reply ui/packages/catalog-ui-search/src/main/webapp/component/query-basic/query-basic.view.js]: i think the only way that could happen is if someone entered an empty string in the catalog ui search config

**Pull request merged**